src/lib/jsonParseV2/parseSubscription.cpp

```diff
@@ -319,6 +320,27 @@ static std::string parseEntitiesVector(ConnectionInfo* ciP, EntityIdVector* eivP
 
         return oe.render(ciP, "");
       }
+      size_t len = type.size();
+      if (len > MAX_ID_LEN)
```

Comment:
This code is used to parse also in the case of update subscription information (PATCH method)?